scripts/engineering/aggregate_phase44d_carry_aware_experimental_family_matrix.py

```
@@ -0,0 +1,421 @@
+    custom_inputs = {
+        "default": args.default_input,
+        "2x2": args.input_2x2,
+        "3x3": args.input_3x3,
+    }
+
+    families_payload: list[dict[str, Any]] = []
+    rows_payload: list[dict[str, Any]] = []
+    canonical_timing: tuple[str, str, str, int] | None = None
+
+    for spec in FAMILY_SPECS:
+        spec = FamilySpec(
```

Ruff (0.15.11)

[warning] `for` loop variable `spec` overwritten by assignment target

(PLW2901)